pnda_env_example.yaml

```diff
@@ -45,6 +52,61 @@ cloud_formation_parameters:
   # Address of gateway for Kakfa interface on public producer subnet
   publicProducerSubnetGateway: 10.0.2.1
 
+openstack_parameters:
+ # keystone_user: Username for the openstack clients to use
+  KEYSTONE_USER: username
+
+  # keystone_user: Password for the openstack clients to use
+  KEYSTONE_PASSWORD: password
+
+  # keystone_tenant: Name of the tenant / project in the openstack environment. The
+  # PNDA stack will be created in this project.
+  KEYSTONE_TENANT: pnda
+
+  # keystone_auth_url: Keystone authentication URL. The Openstack console provides this
+  # under the Access & Security section.
+  KEYSTONE_AUTH_URL: 'http://x.x.x.x:5000/v2.0'
+
+  # keystone_auth_version: Keystone authentication version. The Openstack console provides this
+  # under the Access & Security section.
+  KEYSTONE_AUTH_VERSION: '2'
+
+  # keystone_region_name: Keystone region. The Openstack console provides this
+  # under the Access & Security section.
+  KEYSTONE_REGION_NAME: RegionOne
+
+  # image_id: Base image to use for the created instances. It should be created by
+  # following the guide in https://github.com/pndaproject/pnda-dib-elements
+  #
+  IMAGE_ID: pnda-base
+
+  # CIDR specifying the address range that may access the created PNDA instances
+  whitelistSshAccess: 0.0.0.0/0
+
+  # UUID of the public network in openstack to use
+  PUBLIC_NETWORK_ID: 5982b761-802a-4af3-9c0b-c3b457559179
+
+  # CIDR specifying the address range for the private subnet (PNDA)
+  privateSubnetCidr: 10.0.1.0/24
+
+  # Address of gateway for Kakfa interface on private subnet
+  privateSubnetGateway: 10.0.1.1
+
+  # CIDR specifying the address range for the public producer subnet (Kafka ingress)
+  publicProducerSubnetCidr: 10.0.2.0/24
+
+  # Address of gateway for Kakfa interface on public producer subnet
+  publicProducerSubnetGateway: 10.0.2.1
+
+existing_machines_parameters:
+  #Below 2 variables used in package_install.sh for offline mode
```

Comment:
package-install.sh uses ${vpcCidr}, I don't see privateSubnetCidr or publicProducerSubnetCidr in there.